src/_project-names.ts

```
@@ -0,0 +1,287 @@
+/**
+ * Parse project aliases from environment variable
+ * @returns Map of raw project names to their aliases
+ */
+function getProjectAliases(): Map<string, string> {
+	if (aliasCache !== null) {
+		return aliasCache;
+	}
+
+	aliasCache = new Map();
+
+	const aliasEnv = (process.env[PROJECT_ALIASES_ENV] ?? '').trim();
```

ESLint

[error] Unsafe assignment of an error typed value.

(ts/no-unsafe-assignment)